common/protos/index.ts

```diff
@@ -24,7 +28,9 @@ export interface IProtoClass<IProto, Proto> {
 // meaning that the type of fields cannot be verified; an int can be confused with a string.
 export function verifyObjectMatchesProto<Proto>(
   protoType: IProtoClass<any, Proto>,
-  object: object
+  object: object,
+  suggestReportToDataformTeam: boolean = false,
+  showDocsLink: boolean = true
```

Comment:
give me an `Options` type so that these can be named and therefore readable at the callsite

Reply:
Good shout